doc/functions/dump.rst

```diff
@@ -24,6 +24,14 @@ introspecting its variables:
     Even when enabled, the ``dump`` function won't display anything if the
     ``debug`` option on the environment is not enabled (to avoid leaking debug
     information on a production server).
+    
+    Or defined as following in 
+    # app/config/config.yml
+    services:
+        custom.twig.extension.debug:
+            class: Twig_Extension_Debug
+            tags:
+                 - { name: 'twig.extension' }
```

Comment:
You are documenting Symfony here, which has nothing to do in the Twig documentation